ddtrace/internal/flare/flare.py

```diff
@@ -53,8 +54,9 @@ def __init__(
         self.file_handler: Optional[RotatingFileHandler] = None
         self.url: str = trace_agent_url
         self._api_key: Optional[str] = api_key
+        self.ddconfig = ddconfig
```

Comment:
This is the solution I ended up going with, but it also means I won't be pulling the most recent config (just the config when ddtrace starts up). This was the simplest way to get the config so I went with it, but if we want to be pulling the most recent version, we'll need to do some more involved refactoring.